proto/api/v2/compliance_integration_service.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+
+option go_package = "v2";
+option java_package = "io.stackrox.proto.api.v2";
+
+import weak "google/api/annotations.proto";
+import "api/v2/common.proto";
+
+package v2;
+
+// Next Tag: 6
+message ComplianceIntegration {
+    string id                     = 1;
+    string version                = 2;
+    string cluster_id             = 3;
+    string namespace              = 4;
```

Comment:
Include cluster name in this object. Including the cluster name in the object will save extra roundtrip when clients want to know the cluster name.